### PR TITLE
biiiig music request pr (track batch 4)

### PR DESCRIPTION
**Pull request by sususuyo**
:cl:
- add: Added in many, many songs suggested by the community over the past month
- add: Sam Jones and Space Asshole are now also in the jukebox
- fix: Fixed the volume and format (mono) for Mt. Amient vs Spasm
- fix: Fixed one of the artist names in the jukebox listing (whoops!)

### CODE REVIEW

File: Resources/Audio/Jukebox/attributions.yml [hunk 1]
```diff
@@ -205,3 +205,144 @@
...
+  
+- files: ["music_1.ogg"]
+  license: "Custom"
+  copyright: "Music 1 from P.N.03. Suggested by ghost"
+  source: "https://www.youtube.com/watch?v=hRt9izStuoQ"
+  
+- files: ["overclocked_weakness.ogg"]
+  license: "Custom"
+  copyright: "Overclocked Weakness from Barotrauma. Suggested by Wowls"
+  source: ""
```
Comment (hivehum) [inline Resources/Audio/Jukebox/attributions.yml:272]: forgor a source

File: Resources/Audio/Lobby/attributions.yml [hunk 2]
```diff
@@ -354,3 +354,283 @@
...
+  
+- files: ["music_1.ogg"]
+  license: "Custom"
+  copyright: "Music 1 from P.N.03. Suggested by ghost"
+  source: "https://www.youtube.com/watch?v=hRt9izStuoQ"
+  
+- files: ["overclocked_weakness.ogg"]
+  license: "Custom"
+  copyright: "Overclocked Weakness from Barotrauma. Suggested by Wowls"
+  source: ""
```
Comment (hivehum) [inline Resources/Audio/Lobby/attributions.yml:420]: here as well

File: Resources/Prototypes/Catalog/Jukebox/Standard.yml [hunk 3]
```diff
@@ -228,7 +228,7 @@
 
 - type: jukebox
   id: Impossible
-  name: Darren Knob - Impossible
+  name: Darren Korb - Impossible
```
Comment (hivehum) [inline Resources/Prototypes/Catalog/Jukebox/Standard.yml:231]: ive been thinking of making a pr to fix this for months. thank you.

Comment (sususuyo) [reply Resources/Prototypes/Catalog/Jukebox/Standard.yml:231]: its honestly insanely embarrassing that i did this twice lmao

File: Resources/Audio/Lobby/attributions.yml (hunk 2 above)
Comment (sususuyo) [reply Resources/Audio/Lobby/attributions.yml]: both should be fixed now

**Comment by formlessnameless** [comment]
Each of these track prs adds a loooooooooot of extra data to our server package and we've already had to raise the cap once. I'm going to approve this separately later and try to publish it, because I think it's still well under the cap, but this is probably the last one we should have.

**Comment by sususuyo** [comment]
For sure, I was also worried about how big its gonna be. I think if people keep suggesting tracks I'll probably limit it more and more to one or two per person. I noted down who's suggested what so I can limit that pretty easy! A lot of these tracks in particular are like. 1-2 min long I think so it might not be nearly as big as it might seem also!

**Review by formlessnameless — APPROVED** [review]

**Pull request merged**